### PR TITLE
feat: Add Swagger UI to playground

### LINKED ISSUE #1978: Support full API in playground
The playground should allow users to view and interact with the entire DefraDB API.

### PR DESCRIPTION
**Pull request by nasdf**
## Relevant issue(s)

Resolves #1978 

## Description

This PR updates the playground with a [Swagger UI](https://swagger.io/tools/swagger-ui/) tab that allows users to view and interact with the DefraDB API. The UI is auto generated from the OpenAPI specification served from the `/openapi.json` endpoint.

Screenshot 2023-10-17 at 11 02 12 AM

## Tasks

- [x] I made sure the code is well commented, particularly hard-to-understand areas.
- [x] I made sure the repository-held documentation is changed accordingly.
- [x] I made sure the pull request title adheres to the conventional commit style (the subset used in the project can be found in [tools/configs/chglog/config.yml](tools/configs/chglog/config.yml)).
- [x] I made sure to discuss its limitations such as threats to validity, vulnerability to mistake and misuse, robustness to invalidation of assumptions, resource requirements, ...

## How has this been tested?

Manually

Specify the platform(s) on which this was tested:
- MacOS

### CODE REVIEW

File: playground/src/index.css [hunk 1]
```diff
@@ -18,86 +18,51 @@ html, body, #root {
   height: 100vh;
 }
 
-.graphiql-defradb-plugin {
-  display: flex;
-  flex-direction: column;
-  gap: 16px;
+.swagger-ui .opblock .opblock-summary {
```
Comment (AndrewSisley) [inline playground/src/index.css:21]: question: Is any of this css hand crafted, or is it some default stuff copy-pasted from somewhere?

**Review by AndrewSisley — APPROVED** [review]
LGTM - everything looks to be shaping up nicely :)

**Review by fredcarle — APPROVED** [review]
LGTM! I love the fact that the UI is auto generated for the openapi endpoint response.

File: playground/src/index.css (hunk 1 above)
Comment (nasdf) [reply playground/src/index.css:21]: I went through and manually updated the styles for dark mode. They don't have css variable support in Swagger UI so this was the next best option.

**Pull request merged**